setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+setup(
+    name='visualizer',
+
+    version='0.1.0',
+
+    description='Visualizer component of BIGSEA Asperathos framework',
+
+    url='',
+
+    author='Rafael Falcao',
```

Comment:
Add @javanlacerda ?

Reply:
@javanlacerda Remember to add your name to this document in the future versions.

Reply:
Added!